.github/workflows/python-versions.yml

```diff
@@ -0,0 +1,59 @@
+name: python versions
+on:
+  schedule:
+    # Notifications will be sent to whoever touches this last
+    # You will be able to trigger this workflow manually if need be
+    - cron:  '0 0 * * 0'
```

Comment:
Running once a week is nice, I think this is a good compromise vs. changing on requirements changes. My only recommendation would be to use a better offtime:

```suggestion
    # This runs at 4:00 every Saturday, where GHA should in theory be less busy.
    - cron:  '0 4 * * 6'
```

<img width="501" alt="image" src="https://user-images.githubusercontent.com/14209781/111511234-711b3100-8746-11eb-89a9-6b82a7f9a523.png">